packages/compiler/test/server/completion.test.ts

```diff
@@ -1445,7 +1445,7 @@ describe("identifiers", () => {
         /**
          * my log context
          */
-        context: MyLogContext<string>[];
```

Comment:
Is this change more illustrative? `context` is still allowed in this...context

Reply:
it is a problem with the auto complete that doesn't support those reserved keyword and complete them with backtick so renamed for simplicity here and filed an issue #6756